document intro and outro markers (#253)
mixxxdj/manual: source/chapters/user_interface.rst

             :ref:`control-keyboard` for more information.

.. _interface-intro-outro:

Intro/Outro Markers
-------------------

.. sectionauthor::
   Goodness Chris-Ugari <goodnesschrisugari@yahoo.com>

.. figure:: ../_static/Mixxx-200-Latenight-Deck-intro-outro.png
  :align: center
  :width: 55px
  :figwidth: 100%
  :alt: The Intro/Outro End and Start Markers
  :figclass: pretty-figures

  The Intro/Outro End and Start Markers

You can use the Intro/Outro Markers with Auto DJ to mix in and out of points but unlike hotcues, they do not mark only one point.
The intro and outro are sections and each section is defined by two points. See :ref:`library-auto-dj`.

**Intro Start/End Markers**
 The analyzer places the intro start point at the first sound of a track and you can decide where to place the intro end point.
 Left-clicking on the marker when it is set jumps to the marker. When it is not set, clicking sets the marker to the current play position.
 Right-clicking on the marker when it is set clears the marker.

**Outro Start/End Markers**
The analyzer places the outro end point at the last sound of a track and you can decide where to place the outro start point.

>  Left-clicking on the marker when it is set jumps to the marker. When it is not set, clicking sets the marker to the current play position.

Reviewer: Sphinx build fails with:

```
/opt/build/repo/source/chapters/user_interface.rst:402:Unexpected indentation.
```

Author: Done. Thank you

.. seealso:: You can also use keyboard shortcuts for Cueing. Go to
             :ref:`control-keyboard` for more information.

.. _interface-intro-outro:

Intro/Outro Markers
-------------------

.. sectionauthor::
   RJ Ryan <rryan@mixxx.org>

>    S.Brandt <s.brandt@mixxx.org>

Reviewer 2: Neither of these wrote any part of this.

   Goodness Chris-Ugari <goodnesschrisugari@yahoo.com>

.. figure:: ../_static/Mixxx-200-Latenight-Deck-intro-outro.png
  :align: center
  :width: 55px
  :figwidth: 100%
  :alt: The Intro/Outro End and Start Markers
  :figclass: pretty-figures

  The Intro/Outro End and Start Markers

You can use the Intro/Outro Markers with Auto DJ to mix in and out of points but unlike hotcues, they do not mark only one point.
The intro and outro are sections and each section is defined by two points. See :ref:`library-auto-dj`.

**Intro Start/End Markers**
 The analyzer places the intro start point at the first sound of a track and you can decide where to place the intro end point.

>  Left-clicking on the marker when it is set will jump to the marker and when it is not set, will set the marker to the current play position.

Reviewer 2: ```suggestion
 Left-clicking on the marker when it is set jumps to the marker. When it is not set, clicking sets the marker to the current play position.
```

>  Right-clicking on the marker when it is set, clears the marker.

Reviewer 2: ```suggestion
 Right-clicking on the marker when it is set clears the marker.
```

**Outro Start/End Markers**
The analyzer places the outro end point at the last sound of a track and you can decide where to place the outro start point.

>  Left-clicking on the marker when it is set will jump to the marker and when it is not set, will set the marker to the current play position.

Reviewer 2: Extra space at the beginning of the line

 Right-clicking on the marker when it is set, clears the marker.

> .. note:: If :term:`quantization` is enabled, right-clicking on any of the markers snaps to the nearest beat.

Reviewer 2: This is incorrect. Right clicking clears the marker.

Author: > This is a good start but it doesn't explain what they're useful for. I think that would be too much to cram into the already bloated User Interface chapter, but it should be linked from the User Interface chapter.

Thank you. So where do you suggest it goes?

Reviewer 2: I'm not sure. I was thinking to add a section to the DJing With Mixxx chapter which is kinda a wastebasket of miscellaneous topics that don't fit anywhere else... but maybe there's a better place? What do you think?

Author: I think its usefullness and usage should go in the "Using Auto DJ for Automatic Mixing" section in the Djing With Mixxx chapter, seems like the most suitable place for that right now.

Reviewer 2: What's tricky for the manual is that they're used by AutoDJ but they can also be used without AutoDJ.

Author: Then the "Beatmatching and Mixing" section of the DJing With Mixxx chapter is a good alternative, what do you think?

Reviewer 2: I think a separate section would be better. That section is already long enough. Also, the intro/outro cues are most useful when not beatmatching.

Author: Ok, I'll create a separate section for it in the DJing With Mixxx chapter. Thank you

Author: OK, thank you, was thinking there were more.
